giraffe/src/utils/legend/sort.ts

```diff
@@ -0,0 +1,30 @@
+import {DomainLabel, LineData, LinePosition} from '../../types'
+
+export const getDataSortOrder = (
+  lineData: LineData,
+  rowIndices: number[],
+  position: LinePosition,
+  domainLabel?: DomainLabel
+): number[] => {
+  if (!position || position === 'overlaid') {
+    return rowIndices
+  }
+
+  const domainLabelName = domainLabel || DomainLabel.Y
+
+  const dataMap = {}
+  const measurementValues = Object.keys(lineData).reduce(
+    (accumulator, id) => accumulator.concat(lineData[id][domainLabelName]),
```

Comment:
what are we actually accumulating? seems like `lineData` ? hinting what this is makes it much easier to follow what we're reducing down.

Reply:
Name changed. See the other comment below.